### PR TITLE
Add python dependencies

### PR DESCRIPTION
**Pull request by yanzhaobiomath**
Allow to automatically install the dependent python packages

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -73,3 +73,13 @@ Collate:
     'importPackages.R'
     'plot.R'
 VignetteBuilder: knitr
+
+Config/reticulate:
```
Comment (ClemensKohl) [inline DESCRIPTION:77]: Hi, thanks for adding the python dependencies! I am just unsure if we should install via pip. I am not sure if I remember correctly, but doesnt pip install system-wide also outside of the conda environment?

**Review by ClemensKohl — COMMENTED** [review]
Maybe we should also include
`.onLoad <- function(libname, pkgname) {
  reticulate::[configure_environment](https://rstudio.github.io/reticulate/reference/configure_environment.html)(pkgname)
}`
as suggested here: https://rstudio.github.io/reticulate/articles/python_dependencies.html

File: DESCRIPTION (hunk 1 above)
Comment (ClemensKohl) [reply DESCRIPTION:77]: I think the "[...] PyPI with **pip** [...]" means it is installed with pip from the PyPI repos.

Comment (yanzhaobiomath) [reply DESCRIPTION:77]: yes, that's what I mean. if we set it as FALSE, the packages will be downloaded from the Anaconda repo, which has nothing to do with your concern that "doesn't  pip install system-wide also outside of the conda environment" or not. To answer your question, we may need to make sure with python/conda environment will be used by the user, I guess.

**Pull request closed without merging**